Source/utils/stdcompat/cstddef.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#ifdef __has_include
+#if __has_include(<cstddef>)
+#include <cstddef>
+#if defined(__cplusplus) && __cplusplus >= 201703L
+namespace devilution {
+    using byte = std::byte;
```

Comment:
We do not indent within a namespace, please use clang-format in the future

Reply:
I submitted the changes from the comments to my fork. Should I update it here?

Reply:
Yes you need to make a new pull request since I already merged the original proposal.